src/util/util_creds.h

```diff
@@ -27,7 +27,7 @@
 #include <selinux/context.h>
 #define SELINUX_CTX context_t
 #include <selinux/selinux.h>
-#define SEC_CTX security_context_t
+#define SEC_CTX char *
```

Comment:
If for the safety this should not be something ike:
#define SEC_CTX (char *)

Reply:
No, that would not compile since it is used as a type.

Reply:
(I'm not sure why do we need own 'SEC_CTX' at all but if we do) why not `typedef`?

Reply:
Apparently, it was used to make the code build with and without selinux and to avoid some `#ifdefs`. I changed it to typedef.